src/main/app-updater.ts

```diff
@@ -18,9 +18,9 @@ class NotificationBackchannel {
 
 const title = "Lens Updater";
 
-async function autoUpdateCheck(windowManager: WindowManager): Promise<void> {
+async function autoUpdateCheck(windowManager: WindowManager, args: UpdateInfo): Promise<void> {
   return new Promise(async resolve => {
-    const body = "Install and restart Lens?";
+    const body = `There is an update available (${args.version}). Update and restart Lens?`;
```

Comment:
I think knowing when a release was made public can be useful. Since I am sure some people will make their decision on whether to update based on how old the release is.

```suggestion
    const body = `There is an update (released on ${releaseDate.format("dddd, MMMM Do, yyyy")}) available (${args.version}). Update and restart Lens?`;
```

Reply:
We decided not to include release date here.